imf/tests/test_distributions.py

```diff
@@ -33,3 +34,65 @@ def test_distr():
     ln.pdf(1.)
     ln.cdf(1)
     ln.rvs(1000)
+
+def test_bounds():
+    left,right=1,2
+    tleft, tright=0.5,3
+    ln = D.TruncatedLogNormal(1,1,left,right)
+    assert (ln.pdf(tleft)==0)
+    assert (ln.pdf(tright)==0)
+    assert (ln.cdf(tleft)==0)
+    assert (ln.cdf(tright)==1)
+
+    ln = D.PowerLaw(-3,left,right)
+    assert (ln.pdf(tleft)==0)
+    assert (ln.pdf(tright)==0)
+    assert (ln.cdf(tleft)==0)
+    assert (ln.cdf(tright)==1)
+
+    ln = D.BrokenPowerLaw([-2,-1.1,-3],[left, .6 * left + .3 * right,
+                                            .3 * left + .6 * right, right])
+    assert (ln.pdf(tleft)==0)
+    assert (ln.pdf(tright)==0)
+    assert (ln.cdf(tleft)==0)
+    assert (ln.cdf(tright)==1)
+
+    ln = D.CompositeDistribution([
+        D.TruncatedLogNormal(1,1,left,.75*left+.25*right),
+        D.PowerLaw(-2,.75*left+.25*right,.5*left+.5*right),
+        D.TruncatedLogNormal(1,1,.5*left+.5*right,.25*left+.75*right),
+        D.PowerLaw(-2,.25*left+.75*right,right)
+        ])
+    assert (ln.pdf(tleft)==0)
+    assert (ln.pdf(tright)==0)
+    assert (ln.cdf(tleft)==0)
+    assert (ln.cdf(tright)==1)
+
+
+def test_integral():
+    def checker(x):
+        assert(np.abs(1-x[0])<3*x[1])
```

Comment:
could you add a comment describing what this is testing?  Looks like you're checking that the integral of the PDF over some range is 1 to within measurement error?  (`x[1]` is measurement error in the integrator, right?)

Reply:
Yes, indeed. I've added a comment